core/win32/inject.c

```diff
@@ -1571,7 +1571,7 @@ inject_into_new_process(HANDLE phandle, HANDLE thandle, char *dynamo_path, bool
 #endif
         }
         if (hook_location == 0) {
-            bool target_64 = !x86_code IF_X64(|| DYNAMO_OPTION(inject_x64));
+            bool target_64 = !x86_code;
```

Comment:
Wait: I thought `x86_code` was whether the target executable was 32-bit, and that -inject_x64 wanted to take over in the 64-bit ntdll and so we wanted the original code here?